### DISCUSSION TITLE
The fumadocs ui cli is showing  erros!

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1034] [Q&A]
![image](https://github.com/user-attachments/assets/478cb3cb-eb4a-4782-bfdb-b8d4ffd614dd)

Is this due to react 19 / nextjs 15

**Comment by Gitstar-OC** [comment]
![image](https://github.com/user-attachments/assets/bf69be19-143d-481c-81ff-4b1c6d1a50b2)

It was solved after adding the flags, I think you need to update the version of react and next in the packages.

**Comment by fuma-nama** [comment]
You have installed an invalid react versions

**Comment by Gitstar-OC** [comment]
Hey, thanks I have installed the app with `npx create-next-app@latest` and this is the vesrion the react is showing, should I open an issue in nextjs org ? 
![image](https://github.com/user-attachments/assets/ded28ed1-a8a7-4062-88ec-af90c1ec4ba0)

**Comment by fuma-nama** [comment]
We allowed all versions above 18 including 19, next.js pinned a specific RC version of React which isn't respecting semantic versioning and we don't want to add a RC to peer dep, you can still install react 18 locally, Next.js uses Reaxt 19 regardless of your installed React versions.

**Reply by Gitstar-OC** [reply]
Thanks, I have removed the complete rc thing and just changed the react version to `19.0.0`
